test/TestProjects/DefaultNamespace/readme.md

```diff
@@ -0,0 +1,8 @@
+# DeafultNamespace
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
```

Comment:
The Generate.ps1 script always passes the namespace. https://github.com/Azure/autorest.csharp/blob/feature/v3/eng/Generation.psm1#L29

Try changing the title inside the DefaultNamespace.json to something else.

Reply:
Removed DefaultNamespace as we can not test this behavior for now because we always pass namespace [here  ](https://github.com/Azure/autorest.csharp/blob/feature/v3/eng/Generation.psm1#L29)